js/bitso.js

```diff
@@ -774,6 +932,88 @@ module.exports = class bitso extends Exchange {
         return await this.privateDeleteOrdersOid (this.extend (request, params));
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        if (!Array.isArray (ids)) {
+            throw new ArgumentsRequired (this.id + ' cancelOrders() ids argument should be an array');
+        }
+        let oids = '';
+        for (let i = 0; i < ids.length; i++) {
+            oids += ids[i];
+            if (i < ids.length - 1) {
+                oids += ',';
+            }
+        }
+        const request = {
+            'oids': oids,
+        };
+        const response = await this.privateDeleteOrders (this.extend (request, params));
+        //
+        //     {
+        //         "success": true,
+        //         "payload": ["yWTQGxDMZ0VimZgZ"]
+        //     }
+        //
+        const payload = this.safeValue (response, 'payload', []);
+        const orders = [];
+        for (let i = 0; i < payload.length; i++) {
+            const id = payload[i];
+            orders.push (this.parseCancelledOrder (id));
+        }
+        return orders;
+    }
+
+    parseCancelledOrder (order, market = undefined) {
```

Comment:
Let's move this to parseOrder:

```
let id = undefined;
if (typeof order === 'string') {
    id = order;
} else {
    ...
}
```

A sidenote on naming – we use `canceled` with a single letter `l` throughout the lib. Both types of usage are correct, but we use the one with a single `l` everywhere for consistency.